docs/content/en/docs/install/install.md

```diff
@@ -131,32 +160,25 @@ Configuration options are specified using the format:
 --set key1=value1,key2=value2,....
 ```
 
-Keptn presents a toolkit composed of multiple operators, each of which enables a specific use-case.
-In the next sections we explore basic installation setups based on use-case.
-The installation command is always the following, only the content of the value.yaml file changes:
-
-```shell
-helm repo add keptn https://charts.lifecycle.keptn.sh
-helm repo update
-helm upgrade --install keptn keptn/keptn \
-   --values ./values.yaml \
-   -n keptn-lifecycle-toolkit-system --create-namespace --wait
-```
-
-* The [helm-charts](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/README.md) page
-  contains the full list of available values.
-
 ## Control what components are installed
 
+Keptn presents a toolkit composed of multiple operators,
+each of which enables a specific use-case.
 Keptn v.0.9 and later releases use an umbrella chart
-with separate charts for individual subcharts.
+with separate charts for each operator.
+
+The installation command is always the same;
+only the content of the `value.yaml` file(s) changes:
 
 * Use the Keptn umbrella `values.yaml` file to specify which components are installed
   and to set some global configuration parameters for Keptn
 * Each component has its own `values.yaml` file
   where you set configuration parameters that are specific to that component.
```

Comment:
I don't think this is how users will generally use our chart. They will instead use a single values file to configure everything. I think we should change this info.

Reply:
After discussion in stand-up, I understand and have corrected the text.